daos/pgsql/Database.php

```diff
@@ -227,6 +227,12 @@ public function __construct() {
                         'INSERT INTO version (version) VALUES (12)'
                     ]);
                 }
+                if (strnatcmp($version, '13') < 0) {
+                    \F3::get('db')->exec([
+                        'ALTER TABLE ' . \F3::get('db_prefix') . 'items UNIQUE (uid (150), source);',
```

Comment:
PostgreSQL is smart enough that no length should be necessary. Additionally, I think the syntax is `ALTER TABLE items ADD UNIQUE(uid, source)`, see the definitions for [`ALTER TABLE`](https://www.postgresql.org/docs/current/static/sql-altertable.html) and [`table_constraint`](https://www.postgresql.org/docs/current/static/sql-createtable.html).

Reply:
fix in 7040883bfbd248dae6c5d266912379c31cfcacd5